techniques/general/G65.html

```diff
@@ -186,54 +182,52 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-		<h2>Related Techniques</h2>
+		<h2>関連テクニック</h2>
 		<ul>
 			<li><a href="../general/G63">G63: Providing a site map</a></li>
 			<li><a href="../general/G128">G128: Indicating current location within navigation bars</a></li>
 		</ul>
 	</section>
 <section id="tests">
-		<h2>Tests</h2>
+		<h2>検証</h2>
 		<section class="procedure" id="procedure">
-			<h3>Procedure</h3>
-			<p>When breadcrumb trails have been implemented in a set of web pages:</p>
+			<h3>手順</h3>
+			<p>ェブページ一式の中にパンくずリストが実装されているとき:</p>
 			<ol>
-				<li>Navigate to a web page.</li>
-				<li>Check that a breadcrumb trail is displayed.</li>
-				<li>Check that the breadcrumb trail displays the correct navigational sequence to reach the current location or the correct hierarchical path to the current location within the site structure.</li>
+				<li>あるウェブページにナビゲートする。</li>
+				<li>パンくずリストが表示されていることをチェックする。</li>
+				<li>パンくずリストが、現在位置に達する正しいナビゲーションの順序、又はサイト構造内の現在位置までの正しい階層の経路を表示していることをチェックする。</li>
 				<li>
-					<p>For a breadcrumb trail that does
-						<em>not</em>
-					include the current location:</p>
+					<p>現在位置を含んで<em>いない</em>パンくずリストの場合:</p>
 					<ol>
-						<li>Check that all elements in the breadcrumb trail are implemented as links.</li>
+						<li>パンくずリストのすべての要素がリンクとして実装されていることをチェックする。</li>
 					</ol>
 				</li>
 				<li>
-					<p>For a breadcrumb trail that does include the current location:</p>
+					<p>現在位置を含んでいるパンくずリストの場合:</p>
 					<ol>
-						<li>Check that all elements except for the current location are implemented as links.</li>
-						<li>Check that the current location is not implemented as a link.</li>
+						<li>現在位置を除くすべての要素がリンクとして実装されていることをチェックする。</li>
+						<li>現在位置がリンクとして実装されていないことをチェックする。</li>
 					</ol>
 				</li>
         <li>
-          <p>For a breadcrumb trail that does include the current location and it behaves as a link:</p>
+          <p>現在位置を含み、かつリンクとして動作するパンくずリストの場合:</p>
           <ol>
-						<li>Check that all elements are implemented as links.</li>
-						<li>Check that the current location is programmatically identified as such (e.g., using the <code class="language-html">aria-current</code> attribute).</li>
+						<li>全ての要素がリンクとして実装されていることをチェックする。</li>
+						<li>現在位置がプログラム的に識別されていることをチェックする (例: <code class="language-html">aria-current</code> 属性を使用)。</li>
 					</ol>
         </li>
-				<li>Check that all links navigate to the correct web page as specified by the breadcrumb trail.</li>
+				<li>Cすべてのリンクが、パンくずリストによって指定された正しいウェブページへナビゲートすることをチェックする。</li>
```

Comment:
原文の一部 (Check の C) が残っていました。
```suggestion
				<li>すべてのリンクが、パンくずリストによって指定された正しいウェブページへナビゲートすることをチェックする。</li>
```